devicon.json

```
@@ -3749,6 +3749,31 @@
         "color": "#619cbc",
         "aliases": []
     },
+    {
+        "name": "grpc",
+        "altnames": [
+            "grpcio"
+        ],
+        "tags": [
+            "programming",
+            "c++",
+            "java",
+            "python",
+            "go",
+            "library"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "plain"
+            ]
+        },
+        "color": "#5FC6C5",
```

I forgot to suggest this. Maybe we can change the color from `#5FC6C5` to `#00b0ad`, that's the major color of its gradient.
```suggestion
        "color": "#00b0ad",
```